usefulUtilities/unmuteZone/unmuteZoneClient.js

```diff
@@ -0,0 +1,70 @@
+//
+//  unmuteZoneClient.js
+//
+//  Created by Robin Wilson on 2019-04-08
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+(function() {
+    
+    function onMuteSettingChanged() {
+        // if user mutes themselves in the zone, 
+        // do not apply the previous mute setting
+        usePreviousMuteSetting = false;
+    }
+    
+    
+    function UnmuteZoneClient () {
+        // blank
+    }
+
+
+    var previousMuteSetting = false,
```

Comment:
please add a new `that` variable and make these variables members of that object